ui/desktop/app/styles/app.scss

```diff
@@ -652,3 +652,35 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
     border-radius: 0 0 5px 5px;
   }
 }
+
+// Host Modal
+.host-list-container {
+  border: 1px solid var(--token-color-border-primary);
+  padding: 0.25rem;
+  border-radius: 6px;
+}
+
+.host-container {
+  button {
+    background: unset;
```

Comment:
None blocking comment: what is the purpose/reason we use `unset` here?

Reply:
I wanted the actions and capabilities of a button without it looking like a button so I decided to just unset this style. I can update this to match the background color of the modal if that approach is more clear.

Reply:
No problem, I was just curious.